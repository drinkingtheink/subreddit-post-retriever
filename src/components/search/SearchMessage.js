import React, { Component } from 'react'
import PropTypes from 'prop-types'

class SearchMessage extends Component {
  render() {
    return (
      <div className="search-message">
        <h2>{this.props.headline}</h2>
        <p className="display_name">{this.props.message}</p>
      </div>
    )
  }
}

SearchMessage.propTypes = {
  headline: PropTypes.string,
  message: PropTypes.string
}

export default SearchMessage;