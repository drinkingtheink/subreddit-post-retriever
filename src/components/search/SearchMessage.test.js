import React from 'react';
import ReactDOM from 'react-dom';
import SearchMessage from './SearchMessage';

const compName = 'SearchMessage'

describe(`${compName} =====================`, () => {

  it(`renders without crashing`, () => {
    const div = document.createElement('div');
    ReactDOM.render(<SearchMessage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

});
