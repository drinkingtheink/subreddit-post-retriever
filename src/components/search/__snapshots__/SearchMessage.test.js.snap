// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchMessage ===================== renders correctly with no props 1`] = `ShallowWrapper {}`;

exports[`SearchMessage ===================== should render correctly with no props 1`] = `ShallowWrapper {}`;
